src/db/migrations/20201023172243-create-product.js

```diff
@@ -6,45 +6,43 @@ module.exports = {
         allowNull: false,
         autoIncrement: true,
         primaryKey: true,
-        type: Sequelize.DataTypes.INTEGER,
+        type: Sequelize.INTEGER,
       },
       name: {
-        type: Sequelize.DataTypes.STRING,
         allowNull: false,
+        type: Sequelize.STRING,
       },
       category: {
-        type: Sequelize.DataTypes.STRING,
+        type: Sequelize.STRING,
       },
       description: {
-        type: Sequelize.DataTypes.STRING,
+        type: Sequelize.TEXT,
       },
       price: {
-        type: Sequelize.DataTypes.DOUBLE,
-        allowNull: false,
+        type: Sequelize.DOUBLE,
       },
       image: {
-        type: Sequelize.DataTypes.STRING,
-      },
-      createdAt: {
-        allowNull: false,
-        type: Sequelize.DataTypes.DATE,
-      },
-      updatedAt: {
-        allowNull: false,
-        type: Sequelize.DataTypes.DATE,
+        type: Sequelize.STRING,
       },
       storeId: {
-        type: Sequelize.DataTypes.INTEGER,
         allowNull: false,
+        type: Sequelize.INTEGER,
         references: {
           model: 'Stores',
           key: 'id',
         },
       },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
     });
   },
-
-  down: async queryInterface => {
+  down: async (queryInterface, _Sequelize) => {
```

Comment:
Just checking why _Sequelize parameter needed to be added in?

Reply:
There is no need. I will remove it, thanks!